<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>504</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>504</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>651</width>
    <height>504</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DISTRATO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagens/logo.ico</normaloff>imagens/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 181, 156);</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QComboBox" name="cbPesquisa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Codigo do cliente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Patrimonio</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnPesquisar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>92</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Imagens/Lupa.png</normaloff>../../../Imagens/Lupa.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcluir">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>450</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DISTRATAR</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Imagens/delete-icon.png</normaloff>../../../Imagens/delete-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="gridCliente">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>621</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CNPJ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Endereço</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telefone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Patrimonio</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
